components/brave_new_tab_ui/widgets/shared/styles.ts

```diff
@@ -278,7 +278,7 @@ export const ListItem = styled('li')<StyleProps>`
 export const BackArrow = styled('div')<StyleProps>`
   width: 20px;
   cursor: pointer;
-  margin-left: ${p => p.marketView ? 60 : 0}px;
+  margin-left: ${p => p.marketView ? 137 : 0}px;
```

Comment:
It seems the only place this shared style is used is within the FTX widget. If this were to be a general style for the backarrow as it appears in the only in the header across widgets, styles may look something like this:

```
width: 20px;
cursor: pointer;
position: absolute;
right: 40px;
```

Given that this is also used in the asset detail view, to cover that use case as well, it may be best to nix all css rules besides `width` and `cursor`, and just put the burden of positioning on containers fitting the use case